src/web_app/js/call.js

```diff
@@ -209,6 +287,9 @@ Call.prototype.connectToRoom_ = function(roomId) {
     Promise.all([this.getTurnServersPromise_, this.getMediaPromise_])
         .then(function() {
           this.startSignaling_();
+          if (isChromeApp()) {
+            this.createCleanupQueue_();
```

Comment:
nit: queueCleanupMessages instead, not actually creating something here.
Can you add a comment explaining that we need to queue messages earlier because when window dies only background.js knows?